docs/articles/documentation/test-api/intercepting-http-requests/creating-a-custom-http-request-hook.md

```diff
@@ -21,7 +21,7 @@ You can create your own request hook to handle HTTP requests. This topic describ
     ```js
     import { RequestHook } from 'testcafe';
 
-    class MyRequestHook extends RequestHook {
+    export class MyRequestHook extends RequestHook {
```

Comment:
Not sure about this place, since it's more like pseudo-code here.

But I guess it won't do any damage if we include it here as well.